pygamelib/actuators.py

```diff
@@ -436,6 +459,48 @@ def find_path(self):
                     seen.add((r, c))
         return []
 
+    def __path_find_astar(self):
+        '''
+         This method implements a Breadth First Search algorithm
+        (`Wikipedia <https://en.wikipedia.org/wiki/A*_search_algorithm>`_)
+        to find the shortest path to destination.
+        '''
+
+        queue = PriorityQueue()
+
+        # queue stores a tuple with values:
+        # h - heuristic value = depth + manhattan distance from distance
+        # type(h) = int
+        # path - path to reach current node from start node
+        # type(path) = list
+        # For each node, depth = len(path)
+        
+        initial_h = abs(self.actuated_object.pos[0]-self.destination[0]) + abs(self.actuated_object.pos[1]-self.destination[1]) 
+        
+        queue.put((initial_h,[(self.actuated_object.pos[0], self.actuated_object.pos[1])]))
+        seen = dict()
+        while queue.empty()==False:
```

Comment:
> I'm not sure it's documented

I didn't get this part. I added some comments in the previous lines regarding the content of the queue. So, can you explain here what I need to do, please?

Reply:
Sure, my apologies: the project uses Black to format the code (https://black.readthedocs.io/en/stable/) and flake8 to ensure that the code is compliant with python standards.
I personally use VSCode so it's integrated and I don't even think about it, but you need to install black and run it on your code (without arguments is fine, black is uncompromising anyway).
It is going to reformat your code automatically. 
If your code does not comply with flake8 it will be refused by the CI/CD check (once I fix it to work again...).

Reply:
Yeah, I used Black in VSCode after you requested it.

> I personally use VSCode so it's integrated and I don't even think about it,

I use VSCode too, as my default IDE.



> If your code does not comply with flake8 it will be refused by the CI/CD check (once I fix it to work again...).

It shows "Some Checks haven't completed yet". I don't know if it's because of the formatting.

Pardon my noobness.

I'll be open to even more changes, if you deem any to be necessary.